app/common/commonMenu.js

```diff
@@ -383,6 +383,39 @@ module.exports.aboutBraveMenuItem = () => {
   }
 }
 
+module.exports.braveHomeMenuItem = () => {
+  return {
+    label: locale.translation('home'),
+    accelerator: 'CmdOrCtrl+Shift+H',
+    click: function (item, focusedWindow) {
+      getSetting(settings.HOMEPAGE).split('|').forEach((homepage, i) => {
+        if (i === 0) {
+          appActions.loadURLInActiveTabRequested(getCurrentWindowId(), homepage)
+        } else {
+          appActions.createTabRequested({
+            url: homepage,
+            windowId: getCurrentWindowId()
+          })
+        }
+      })
+    }
+  }
+}
+
+module.exports.reopenLastClosedWindowMenuItem = () => {
```

Comment:
@bradleyrichter this doesn't seem working. But code is same as what's there in for [menu.js](https://github.com/brave/browser-laptop/blob/master/app/browser/menu.js#L318l)
Can i have some input on this. Thanks